scene/gui/color_picker.cpp

```diff
@@ -159,7 +159,10 @@ void ColorPicker::_html_entered(const String &p_html) {
 	if (updating)
 		return;
 
-	color = Color::html(p_html);
+	float last_alpha = color.a;
+  	color = Color::html(p_html);
```

Comment:
You seem to have a mix of tabs and spaces, indentation should be tab-based only.
Can you fix this with `git commit --amend` and update the PR with `git push --force`?